tools/meteor-services/auth.js

```diff
@@ -65,6 +65,9 @@ var loggedInAccountsConnection = function (token) {
     connection.close();
 
     if (err.error === 403) {
+      // Better than saying nothing at all
+      Console.error("warning: login token was rejected by the server");
```

Comment:
@dhrubins Any updates on this?  This PR is appreciated and this sounds bizarre and annoying enough to track down that this would likely be really nice to get a fix in for if you have some time.